SpeechToTextV1.playground/Pages/Custom grammars.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,71 @@
+//:## Custom words
+
+import PlaygroundSupport
+
+// Enable support for asynchronous completion handlers
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+import SpeechToTextV1
+
+let speechToText = setupSpeechToTextV1()
+let customizationID = "{customization-id}"
+let grammarName = "list-abnf"
+
+//:### Add a grammar
+
+let grammarFile = Bundle.main.url(forResource: "list", withExtension: "abnf")!
+speechToText.addGrammar(
+    customizationID: customizationID,
+    grammarName: grammarName,
+    grammarFile: grammarFile.absoluteString,
+    contentType: "application/srgs")
+{
+    _, error in
+
+    if let error = error {
+        print(error.localizedDescription)
+        return
+    }
+
+    print("successfully added grammar")
```

Comment:
For these methods that give no response, I have started using the description from the 200 status code as the message text in this print. This is a _very minor_ thing, but gives a nice linkage back to the docs. So in this case I would use
```
print("The request succeeded.")
```
I'll leave it up to you to decide if you want to follow that convention.

Reply:
I still don't understand how a message like "The request succeeded." would be better.

Reply:
I don't disagree, but it's more a matter of treating the API doc as the "single source of truth".  If the description in the API doc is bad, we should fix it there.

But we can pursue this on a separate track -- it shouldn't hold up this work.